.github/actions/install-go/action.yml

```diff
@@ -17,6 +17,6 @@ runs:
     - name: Set up Go
       uses: actions/setup-go@v5
       with:
-        go-version: '1.23.7'
+        go-version-file: go.mod
```

Comment:
Ooh this is nice https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file